manifests/testing/my-ovm.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kubevirt.io/v1alpha1
```

Comment:
we've been putting our sample manifests for VMs under the cluster/* directory.

Arguably, a manifests/samples/ directory might make the most sense for OVM/VM manifests.

Reply:
I like the idea of having the manifest/samples folder. It makes finding the examples easier.

Reply:
Lets tackle a common example/sample directory in a different PR. @petrkotas could you move the ovm demo yaml to /cluster right now?